### PR TITLE
Fix todo & any & minor lint

### PR DESCRIPTION
**Pull request by vglfr**

### CODE REVIEW

File: client/src/containers/ProfileContainer/ProfileContainer.test.tsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import { ProfileContainer } from './index';
+import * as p from '../../components/Profile';
```
Comment (InnaKutsik) [inline client/src/containers/ProfileContainer/ProfileContainer.test.tsx]: з ---> profile

File: server/src/controllers/auth.controllers/test/login.test.ts [hunk 2]
```diff
@@ -15,25 +15,25 @@ describe('User login controller:', () => {
         };
     });
 
-    test.skip('login - failure', async () => {
+    test('user does not exist - failure', async () => {
         const mockUserExist = jest.spyOn(ue, 'userExist');
-        mockUserExist.mockReturnValue(new Promise((res: any): Promise<null> => res(null))); // FIXME any
+        mockUserExist.mockReturnValue(new Promise<null>(res => res(null))); // tslint:disable-line:typedef
```
Comment (InnaKutsik) [inline server/src/controllers/auth.controllers/test/login.test.ts]: why disable?

File: server/src/controllers/auth.controllers/test/login.test.ts [hunk 3]
```diff
@@ -15,25 +15,25 @@ describe('User login controller:', () => {
...
 
-    test('generic authentication error happened', async () => {
+    test('generic authentication error - failure', async () => {
         mockingoose(User).toReturn({isVerified: true}, 'findOne');
-        const fakeUser = await User.findOne({});
+        const fakeUser: IUserModel = await User.findOne({}) as IUserModel;
 
         const mockUserExist = jest.spyOn(ue, 'userExist');
-        mockUserExist.mockReturnValue(new Promise((res: any): Promise<null> => res(fakeUser))); // FIXME any
+        mockUserExist.mockReturnValue(new Promise<IUserModel>(res => res(fakeUser))); // tslint:disable-line:typedef
```
Comment (InnaKutsik) [inline server/src/controllers/auth.controllers/test/login.test.ts]: why disabled

File: server/src/controllers/confirm.user.controllers/test/confirm.token.controller.test.ts [hunk 4]
```diff
@@ -12,19 +12,22 @@ const fakeNext = jest.fn(() => { /* */ });
...
-        const fakeToken = await Token.findOne({});
+        const fakeToken: ITokenModel = await Token.findOne({}) as ITokenModel;
 
         mockingoose(User).toReturn({}, 'findOne');
-        const fakeUser = await User.findOne({});
+        const fakeUser: IUserModel = await User.findOne({}) as IUserModel;
 
         const mockIsTokenExist = jest.spyOn(tr, 'isTokenExist');
-        mockIsTokenExist.mockReturnValue(new Promise((res: any): Promise<ITokenModel> => res(fakeToken))); // FIXME any
+        const value1 = new Promise<ITokenModel>(res => res(fakeToken)); // tslint:disable-line:typedef
```
Comment (InnaKutsik) [inline server/src/controllers/confirm.user.controllers/test/confirm.token.controller.test.ts]: why disabled?

File: client/src/containers/ProfileContainer/ProfileContainer.test.tsx (hunk 1 above)
Comment (vglfr) [reply client/src/containers/ProfileContainer/ProfileContainer.test.tsx]: Fixed here & other similar.

File: server/src/controllers/auth.controllers/test/login.test.ts (hunk 2 above)
Comment (vglfr) [reply server/src/controllers/auth.controllers/test/login.test.ts]: Rewrote here & similar.

File: server/src/controllers/auth.controllers/test/login.test.ts (hunk 3 above)
Comment (vglfr) [reply server/src/controllers/auth.controllers/test/login.test.ts]: Ibid top.

File: server/src/controllers/confirm.user.controllers/test/confirm.token.controller.test.ts (hunk 4 above)
Comment (vglfr) [reply server/src/controllers/confirm.user.controllers/test/confirm.token.controller.test.ts]: Ibid top.

**Pull request merged**